compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -98,6 +98,10 @@
     JOIN_KEYWORD(244, "join"),
     OUTER_KEYWORD(245, "outer"),
     EQUALS_KEYWORD(246, "equals"),
+    ORDER_KEYWORD(247, "order"),
+    BY_KEYWORD(248, "by"),
+    ASCENDING_KEYWORD(249, "ascending"),
+    DESCENDING_KEYWORD(250, "descending"),
```

Comment:
theres an overlap in the numbers (another set of keywords starts with 250) - shall we update the "Type keywords" (the ones that are below) to start with 300?